Python/Product/EnvironmentsList/Resources.resx

```diff
@@ -123,6 +123,40 @@
   <data name="ConfigurationExtensionHelpContent" xml:space="preserve">
     <value>These environment settings may be configured. The tooltips on each field include more information and examples.</value>
   </data>
+  <data name="CondaExtensionDisplayName" xml:space="preserve">
+    <value>Create</value>
+  </data>
+  <data name="CondaExtensionHelpContent" xml:space="preserve">
+    <value>These environment settings may be configured. The tooltips on each field include more information and examples.</value>
+  </data>
+  <data name="CondaExtensionCreateLabel" xml:space="preserve">
+    <value>_Create</value>
+  </data>
+  <data name="CondaExtensionCreateTooltip" xml:space="preserve">
+    <value>Create the environment.</value>
+  </data>
+  <data name="CondaExtensionEnvironmentNameLabel" xml:space="preserve">
+    <value>Name</value>
+  </data>
+  <data name="CondaExtensionEnvironmentNameHelp" xml:space="preserve">
+    <value>The name of the new environment.</value>
+  </data>
+  <data name="CondaExtensionEnvironmentNameWatermark" xml:space="preserve">
+    <value>e.g. My New Environment</value>
```

Comment:
Should we offer "env" as the example? I think that's more common.

Reply:
sure